docs/templates/eia923_child.rst.jinja

```diff
@@ -66,26 +64,30 @@ both the Form and the Form Instructions files.
 
 {% block notable_irregularities %}
 
-**File Naming Conventions**
+File Naming Conventions
+-----------------------
```

Comment:
i'm a lil surprised to see this and some other little docs cleanup stuff in this pr. its all good but ⏲️

Reply:
I didn't bill for a lot of this peripheral stuff. It just came up with trying to get rid of the dead links for tables that aren't written to the DB because of the move to more parquet-only tables.